docs/Changelog.md

```diff
@@ -14504,3 +14504,66 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain input and output types to high-precision and 8 bit numeric tensors.</dd>
 </dl>
 
+### <a name="SoftmaxCrossEntropyLoss-12"></a>**SoftmaxCrossEntropyLoss-12**</a>
+
+  Loss function that measures the softmax cross entropy
+  between 'scores' and 'labels'.
+  The loss can be described as:
+      L = (l_1, l_2, ..., l_N), where N is the batch_size
+  
+  shape(scores): (N, C) where C is the number of classes, or (N, C, d1, d2,..., dk),
```

Comment:
```suggestion
  shape(scores): (N, C) where C is the number of classes, or (N, C, D1, D2,..., Dk),
```
because we have `y[i][c][d1][d2]..[dk]` below.